templates/legal/websites.html

```diff
@@ -9,138 +9,150 @@
         </li>
         <li class="p-list__item">
-          <a href="https://cdimages.ubuntu.com">cdimages.ubuntu.com</a>
+          <a class="p-link--external" href="https://canonical.com">canonical.com</a>
         </li>
         <li class="p-list__item">
-          <a href="https://certification.ubuntu.com">certification.ubuntu.com</a>
+          <a class="p-link--external" href="https://cdimages.ubuntu.com">cdimages.ubuntu.com</a>
         </li>
         <li class="p-list__item">
-          <a href="https://cloud-images.ubuntu.com">cloud-images.ubuntu.com</a>
+          <a class="p-link--external" href="https://certification.ubuntu.com">certification.ubuntu.com</a>
         </li>
         <li class="p-list__item">
-          <a href="https://cloud-init.io">cloud-init.io</a>
+          <a class="p-link--external" href="https://charmed-osm.com">charmed-osm.com</a>
         </li>
         <li class="p-list__item">
-          <a href="https://cn.ubuntu.com">cn.ubuntu.com</a>
+          <a class="p-link--external" href="https://charmhub.io">charmhub.io</a>
         </li>
         <li class="p-list__item">
-          <a href="https://discourse.ubuntu.com">discourse.ubuntu.com</a>
+          <a class="p-link--external" href="https://cloud-images.ubuntu.com">cloud-images.ubuntu.com</a>
         </li>
         <li class="p-list__item">
-          <a href="https://design.ubuntu.com">design.ubuntu.com</a>
+          <a class="p-link--external" href="https://cloud-init.io">cloud-init.io</a>
         </li>
         <li class="p-list__item">
-          <a href="https://discourse.juju.is">discourse.juju.is</a>
+          <a class="p-link--external" href="https://cn.ubuntu.com">cn.ubuntu.com</a>
         </li>
         <li class="p-list__item">
-          <a href="https://discourse.maas.io">discourse.maas.io</a>
+          <a class="p-link--external" href="https://design.ubuntu.com">design.ubuntu.com</a>
         </li>
         <li class="p-list__item">
-          <a href="https://discourse.ubuntu.com">discourse.ubuntu.com</a>
+          <a class="p-link--external" href="https://discourse.ubuntu.com">discourse.ubuntu.com</a>
         </li>
         <li class="p-list__item">
-          <a href="https://docs.snapcraft.io">docs.snapcraft.io</a>
+          <a class="p-link--external" href="https://discourse.juju.is">discourse.juju.is</a>
         </li>
         <li class="p-list__item">
-          <a href="https://docs.ubuntu.com">docs.ubuntu.com</a>
+          <a class="p-link--external" href="https://discourse.maas.io">discourse.maas.io</a>
         </li>
         <li class="p-list__item">
-          <a href="https://dqlite.io">dqlite.io</a>
+          <a class="p-link--external" href="https://discourse.ubuntu.com">discourse.ubuntu.com</a>
         </li>
         <li class="p-list__item">
-          <a href="https://errors.ubuntu.com">errors.ubuntu.com</a>
+          <a class="p-link--external" href="https://docs.snapcraft.io">docs.snapcraft.io</a>
```

Comment:
This one is just a redirect to snapcraft itself now.

Reply:
fixed